BenchmarkVCFs/SimpleBenchmark.wdl

```diff
@@ -0,0 +1,596 @@
+
+    # Add in default "Whole Genome" empty interval list
+    Array[File] full_strat_intervals = flatten([[""], strat_intervals])
+    Array[String] full_strat_labels = flatten([[""], strat_labels])
+
+    # Make Stratifier objects
+    scatter (strat in zip(full_strat_labels, full_strat_intervals)) {
+        Stratifier stratifier_list = {"label": strat.left, "interval": if strat.right != "" then strat.right else NULL_FILE}
+    }
+
+    # Make VariantSelector objects, with default None, Het, and HomVar types
+    Array[String] default_bcf_selectors =  ["", "GT='het'", "GT='hom'"]
+    Array[String] default_bcf_labels = ["", "Het", "HomVar"]
+    Array[String] full_bcf_selectors = flatten([default_bcf_selectors, bcf_selectors])
+    Array[String] full_bcf_labels = flatten([default_bcf_labels, bcf_labels])
+    scatter (selection in zip(full_bcf_labels, full_bcf_selectors)) {
+        VariantSelector selector_list = {"bcf_label": selection.left, "bcf_selector": selection.right}
+    }
+
+    # Perform analysis over all Stratifiers
+    scatter (stratifier in stratifier_list) {
+        call SubsetVCF as SubsetEval {
+            input:
+                input_vcf=call_vcf,
+                input_vcf_index=call_vcf_index,
+                input_sample_name=call_vcf_sample_name,
+                reference=reference,
+                stratifier=stratifier,
+                interval_padding=interval_padding,
+                gatk_tag=subset_gatk_tag,
+                preemptible=3
+        }
+
+        call SubsetVCF as SubsetTruth {
+            input:
+                input_vcf=base_vcf,
+                input_vcf_index=base_vcf_index,
+                input_sample_name=base_vcf_sample_name,
+                reference=reference,
+                stratifier=stratifier,
+                interval_padding=interval_padding,
+                gatk_tag=subset_gatk_tag,
+                preemptible=3
+        }
+
+        # Run over different score_fields to produce other ROC outputs
+        scatter (score_field in score_fields) {
+            call VCFEval as StandardVCFEval {
+                input:
+                    call_vcf=SubsetEval.output_vcf,
+                    call_vcf_index=SubsetEval.output_vcf_index,
+                    call_output_sample_name=call_output_sample_name,
+                    base_vcf=SubsetTruth.output_vcf,
+                    base_vcf_index=SubsetTruth.output_vcf_index,
+                    base_output_sample_name=base_output_sample_name,
+                    reference=reference,
+                    evaluation_bed=evaluation_bed,
+                    score_field=score_field,
+                    strat_label=stratifier.label,
+                    preemptible=3
+            }
+        }
+
+        scatter (selector in selector_list) {
+            # Use the first outputs for tp_base_vcf, fp_vcf, etc. since these should be the same across all score_fields
+            call BCFToolsStats {
```

Comment:
I thought there was a problem with vcfeval removing all the vcf annotations, so they couldn't be used for selecting?  Or does "split" mode not have that issue?

Reply:
The split mode definitely doesn't have this problem! That's probably the main reason for the switch to doing it this way. Basically the e.g. `fn.vcf.gz`, etc. files are pure subsets of the original VCFs.